docs/development.md

```diff
@@ -57,15 +57,7 @@ As part of the environment, a special python executable is generated in the
 `bin/` subdirectory.  That executable is preconfigured to use the correct
 `PYTHONPATH` to pick up the dependencies in the `_env/` directory.
 
-To build the code, you will need to make the environment, then run the
-set up scripts. This is done by the following commands (run from the top level
-Ellis directory):
-
-    make env
-    _env/bin/easy_install --allow-hosts=None -f eggs/ eggs/*.egg
-    _env/bin/python setup.py install
-    cd common
-    ../_env/bin/python setup.py install
+To build the code, run `make env` in the top level Ellis directory.
```

Comment:
While you're here, line 52 above says 'uses virtualenv and buildout' - I don't think we use buildout any more.